pyhf/__init__.py

```diff
@@ -390,7 +290,7 @@ def logpdf(self, pars, data):
         cut = int(data.shape[0]) - len(self.config.auxdata)
         actual_data, aux_data = data[:cut], data[cut:]
         lambdas_data = self.expected_actualdata(pars)
-        summands = tensorlib.log(_poisson_impl(actual_data, lambdas_data))
+        summands = tensorlib.log(tensorlib.poisson(actual_data, lambdas_data))
```

Comment:
a rebase to master should get equivalent changes in. see #122

Reply:
done.